userguide/tutorials/what_is_kill_bill.adoc

```diff
@@ -65,52 +65,55 @@ To know more, follow our https://docs.killbill.io/latest/userguide_deployment.ht
 
 === Kill Bill Platform
 
-We refer to Kill Bill as a *platform* because of its extensibility: You can write plugins to **modify the behavior of the system** or to **integrate with third-party vendors**, such as fraud detection solutions (Accertify, Feedzai, etc.) or Tax providers (Avalara AvaTax, etc.). We offer some open-source plugins on our http://github.com/killbill/killbill[GitHub organization], but in order to match your exact business requirements you may need to write your own.
+We refer to Kill Bill as a *platform* because of its extensibility: You can write plugins to **modify the behavior of the system** or to **integrate with third-party vendors**, such as fraud detection solutions (Accertify, Feedzai, etc.) or Tax providers (Avalara AvaTax, Vertex, etc.). We offer some open-source plugins on our http://github.com/killbill/killbill[GitHub organization], but in order to match your exact business requirements, you may need to write your own.
```

Comment:
```suggestion
We refer to Kill Bill as a *platform* because of its extensibility: You can write plugins to **modify the behavior of the system** or to **integrate with third-party vendors**, such as fraud detection solutions (Accertify, Feedzai, etc.) or tax providers (Avalara AvaTax, Vertex, etc.). We offer some open-source plugins on our http://github.com/killbill/killbill[GitHub organization], but in order to match your exact business requirements, you may need to write your own.
```